.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,33 @@
+name: Lint and Test Charts
```

Comment:
The descheduler project recently setup all CI tests using [k8s prow](https://github.com/kubernetes/test-infra/tree/master/prow). The Travis CI configuration will probably be removed soon. Right now I don't think we should start using GitHub actions to run CI tests for the helm chart.

My recommendation is to remove this file from this PR. We can figure out how to add helm chart linting through prow in a follow up PR.

Reply:
I'd strongly recommend against dropping the lint and test component of the Helm integration. As with the tagging strategy below I think it should be seen as a separate but complimentary process with a separate pipeline. In this scenario using GitHub Actions for both releases and testing would make sense.

Reply:
@stevehipwell thanks for your feedback and thoughts. My opinion is still that the descheduler project should not three separate CI systems(Travis, Prow, and GH Actions). We are trying to get all CI tests run through Prow, so that we have a single CI system.

I think this file should be removed from this PR.

If you want to you can update the `Makefile` in this PR to add some targets for helm chart linting and testing. But this is not required before merging in my opinion.

Reply:
@seanmalloy I'll comment out the `lint-test` GH Action, although we will still be using all three CI systems as GH Actions is still being used for releasing. In the scope of this chart I don't think that the automated tests would have too many benefits, so I agree they could be omitted for now.